main.go

```diff
@@ -116,6 +120,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&machinePoolConcurrency, "machinepool-concurrency", 10,
 		"Number of machine pools to process simultaneously")
 
+	fs.IntVar(&clusterResourceSetConcurrency, "clusterresourceset-concurrency", 1,
```

Comment:
I would probably start with something greater than one here. Any objections to 10?

Reply:
@fabriziopandini had a comment above:

```
Is it necessary to handle concurrency for this controller? with default 10?
it seems to me overkilling for a controller that gets triggered once in a lifetime of each cluster.
```

Reply:
My original intent was to simplify and get rid of the flag, but if we are keeping the flag, I'm fine with any value